apps/dashboard/app/new/page.tsx

```diff
@@ -79,45 +80,68 @@ export default async function (props: Props) {
-            <div className="flex items-center justify-center p-4 border rounded-lg bg-primary/5">
-              <GlobeLock className="w-6 h-6 text-primary" />
+          <div className="flex flex-col gap-4 lg:gap-10 p-8 duration-200 border rounded-lg border-border hover:border-primary justify-between">
+            <div className="flex flex-col gap-4">
+              <div className="flex items-center justify-center p-4 border rounded-lg bg-primary/5">
+                <GlobeLock className="w-6 h-6 text-primary" />
+              </div>
+              <h4 className="text-lg font-medium">I want to ratelimit something</h4>
+              <p className="text-sm text-content-subtle">
+                Global low latency ratelimiting for your application.
+              </p>
+              <ol className="ml-2 space-y-1 text-sm list-disc list-outside text-content-subtle">
+                <li>Low latency</li>
+                <li>Globally consistent</li>
+                <li>Powerful analytics</li>
+              </ol>
             </div>
-            <h4 className="text-lg font-medium">I want to ratelimit something</h4>
-            <p className="text-sm text-content-subtle">
-              Global low latency ratelimiting for your application.
-            </p>
-            <ol className="ml-2 space-y-1 text-sm list-disc list-outside text-content-subtle">
-              <li>Low latency</li>
-              <li>Globally consistent</li>
-              <li>Powerful analytics</li>
-            </ol>
             <Link href={`/new?workspaceId=${workspace.id}&product=ratelimit`}>
               <Button variant="primary" type="button" className="w-full">
                 Create Ratelimit
               </Button>
             </Link>
           </div>
+          <div className="flex flex-col gap-4 lg:gap-10 p-8 duration-200 border rounded-lg border-border hover:border-primary justify-between">
+            <div className="flex flex-col gap-4">
+              <div className="flex items-center justify-center p-4 border rounded-lg bg-primary/5">
+                <DatabaseZap className="w-6 h-6 text-primary" />
+              </div>
+              <h4 className="text-lg font-medium">I want to cache an LLM</h4>
+              <p className="text-sm text-content-subtle">
+                Faster, cheaper LLM API calls through re-using semantically similar previous
+                responses.
+              </p>
+              <ol className="ml-2 space-y-1 text-sm list-decimal list-outside text-content-subtle">
+                <li>You switch out the baseUrl in your requests to OpenAI with your gateway URL</li>
+                <li>Unkey will automatically start caching your responses</li>
+                <li>Monitor and track your cache usage here</li>
+              </ol>
+            </div>
+            <CreateSemanticCacheButton />
+          </div>
```

Comment:
Depending on if you take my feedback on the we should allow for input, this may need to move to the form input section and copy surrounding generalized LLM caching could go here.